pkg/render/manager.go

```diff
@@ -609,6 +591,11 @@ func managerClusterRole(managementCluster, managedCluster, openshift bool) *rbac
 				Resources: []string{"serviceaccounts", "namespaces"},
 				Verbs:     []string{"list"},
 			},
+			{
```

Comment:
Could you please add a comment explaining exactly how these permissions are used?

I know we failed to do that for some of the other ones, but it's really important to document what permissions we're providing and why they are required.